xml/deployment_sysreqs.xml

```diff
@@ -31,6 +31,25 @@
    The minimum supported cluster size is four nodes: a single &admin_node;, one
    &master_node;, and two &worker_node;s.
   </para>
+  <important>
+   <title>Cluster Update Reserves</title>
+   <para>
+    You should always size your cluster to accomodate your expected workloads and
+    contingency resources for updates. During updates, worker nodes will become
+    temporarily unavailable to process workloads and their assigned load must
+    be shifted somewhere else in the cluster. In very small clusters with only
+    few worker nodes this is sometimes not possible (e.g. two workers each with
+    51% utilization).
```

Comment:
Not sure about the example TBH, this case is assuming two workers of the same size and without specified can be misleading, since if you have two workers of very different sizes even if the biggest is at 20% can lead to trouble (depending on the difference between them). I would omit the example.

Reply:
ok will remove that example